python/IECore/ClassLoader.py

```diff
@@ -215,7 +215,7 @@ def defaultLoader( cls, envVar ) :
 			else :
 				msg( Msg.Level.Warning, "ClassLoader.defaultLoader", "Environment variable %s not set." % envVar )
 
-			loader = cls( SearchPath( os.path.expandvars( sp ) ) )
+			loader = cls( SearchPath( os.path.expandvars( sp ), os.pathsep ) )
```

Comment:
Is this definitely necessary? I think it should have been covered already by 49c6d73ff8dbe49748ebf673230ae1f680868b6f.

Reply:
Ah, wonderful, I hadn't seen that commit. Building and testing now without that explicit os.pathsep.